dockerfiles/sv-base/install_deprecated_R_package.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu -o pipefail
+
+MODULE_ARCHIVE_URL=$1
+
+ARCHIVE_DIR=$(mktemp -d "${TMPDIR:-/tmp/}$(basename $0).XXXXXXXXXXXX")
+trap "rm -rf $ARCHIVE_DIR" EXIT
+
+MODULE_ARCHIVE_DEST="$ARCHIVE_DIR/$(basename "$MODULE_ARCHIVE_URL")"
+curl "$MODULE_ARCHIVE_URL" --output "$MODULE_ARCHIVE_DEST"
+Rscript -e "install.packages('$MODULE_ARCHIVE_URL', repos = NULL, quiet = TRUE)"
```

Comment:
Do you want the first parameter to be `$MODULE_ARCHIVE_DEST` instead of `$MODULE_ARCHIVE_URL`? I think this will make R re-download it from the URL, won't it?

Reply:
Chris, I made more typos in this script than it has lines, by like a factor of two. And yet, here's another...
Good catch, I fixed it.